src/Apps/Artist/Components/ArtistRecommendations.tsx

```diff
@@ -0,0 +1,57 @@
+import { Serif } from "@artsy/palette"
+import { ArtistApp_artist } from "__generated__/ArtistApp_artist.graphql"
+import { ArtistRecommendations_artist } from "__generated__/ArtistRecommendations_artist.graphql"
+import { ArtistRecommendationsQuery } from "__generated__/ArtistRecommendationsQuery.graphql"
+import { SystemContext } from "Artsy"
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+import React, { useContext } from "react"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+
+interface ArtistRecommendationsProps {
+  artist: ArtistRecommendations_artist
+}
+
+export const ArtistRecommendations: React.FC<
+  ArtistRecommendationsProps
+> = props => {
+  const {
+    artist: { name },
+  } = props
+
+  return (
+    <div>
+      <Serif size="8" color="black100">
+        Related to {name}
+      </Serif>
+    </div>
+  )
+}
+
+export const ArtistRecommendationsFragmentContainer = createFragmentContainer(
+  ArtistRecommendations,
+  graphql`
+    fragment ArtistRecommendations_artist on Artist {
+      name
+    }
+  `
+)
+
+export const ArtistRecommendationsQueryRenderer: React.FC<{
```

Comment:
@mzikherman Let us know if this looks different than you'd have expected!

Reply:
Yep looks great!